File: app/src/main/java/com/neome/api/ent/session/sig/SigTopicSpreadsheetRef.java
// neome.ai API - do not change
//

package com.neome.api.ent.session.sig;

import com.neome.api.core.session.sig.SigTopic;
import com.neome.api.meta.base.Types.MetaIdSpreadsheetRef;
import com.neome.api.meta.base.Types.RowId;

@SuppressWarnings("unused")
public class SigTopicSpreadsheetRef extends SigTopic
{
  public MetaIdSpreadsheetRef metaIdSpreadsheetRef;

  public RowId targetRowId;
}

File: app/src/main/java/com/neome/api/ent/session/sig/SigTopicSpreadsheetRef.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.ent.session.sig

import com.neome.api.meta.base.Types.MetaIdSpreadsheetRef
import com.neome.api.meta.base.Types.RowId
import com.neome.api.core.session.sig.SigTopic

interface SigTopicSpreadsheetRef : SigTopic
{
  val metaIdSpreadsheetRef: MetaIdSpreadsheetRef
  val targetRowId: RowId
}